test/unit/test_server.py

```diff
@@ -34,8 +34,7 @@
 
 
 def test_api_compatible():
-    '''
-    Test whether `swift_scality_backend.server.app_factory`'s result is
+    '''Test whether `swift_scality_backend.server.app_factory`'s result is
```

Comment:
I don't like this rule either: dura lex, sed lex

In most case it makes sense though, so I don"t want to make an exception and I can live with this weird doc string here.